Modules/_sqlite/connection.c

```diff
@@ -556,8 +556,6 @@ _pysqlite_set_result(sqlite3_context* context, PyObject* py_val)
     } else if (PyObject_CheckBuffer(py_val)) {
         Py_buffer view;
         if (PyObject_GetBuffer(py_val, &view, PyBUF_SIMPLE) != 0) {
-            PyErr_SetString(PyExc_ValueError,
```

Comment:
Looks like there's no test for this change.

Reply:
I'll add one, thanks!

Reply:
Added in b4a95bdb096d379bc1698778a1506537224ea660